docs-src/tut-7-rpc.scrbl

```diff
@@ -340,11 +230,22 @@ Alice went from 10 to 4.9999
   Bob went from 10 to 14.9999
 }
 
+@margin-note{
+What is this @tt{"venv"} thing?
+
+A Python
+@link["https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment"]{venv}
+is a "virtual environment" that sandboxes dependencies to avoid cluttering your
+system directories.
+}
+
 You may replay as many times as you would like by pressing @tt{"up"} once then
 @tt{"return"} on your keyboard in this terminal.
 
-When you are done type @tt{"control+c"} and then @tt{./reach down} in the first
-terminal.
+When you are done type @tt{"control+c"}, then @tt{"./reach down"} in the first
+terminal, and then @tt{"deactivate"} in the second.
```

Comment:
Does this `deactive` mean `./reach deactivate`?

Reply:
`deactivate` is a shell alias that's created when you enter a `venv` - it's basically how you clear the `venv`'s overrides and exit the virtual environment once done.

This was another case where I thought explanation might run counter to the Divio advice, but maybe it deserves a margin note? Or perhaps a sentence in the existing "What's this `venv` thing?" sidebar?

Reply:
Ah, that makes sense

Reply:
Resolved in 0289e8495254ef2b2a1f33c567422bc1fade9a57.